shared/src/business/useCases/users/generateChangeOfAddress.test.js

```diff
@@ -73,21 +78,25 @@ describe('generateChangeOfAddress', () => {
   };
 
   beforeEach(() => {
+    mockCase = mockCaseWithPrivatePractitioner;
```

Comment:
is there any worry of mutating `mockCaseWithPrivatePractitioner` in one of these tests, and this wouldn't be the same thing the next time we made this call?

Reply:
Good catch. It looks like some of the tests are modifying `mockCaseWithPrivatePractitioner`, we will go and update those so they modify `mockCase` instead!